frontend/src/metabase/query_builder/components/NativeQueryEditor.css

```diff
@@ -4,6 +4,7 @@
   border-radius: 4px;
   background-color: white;
   color: #4c5773;
+  width: 400px;
```

Comment:
Model / question names can be longer than tables and columns, so we need more space in the autocomplete component to display them.